pkg/apiserver/registry/security/workloadidentity/storage/storage.go

```diff
@@ -33,7 +33,11 @@ func NewStorage(
 	tokenIssuer workloadidentityutils.TokenIssuer,
 	coreInformerFactory gardencoreinformers.SharedInformerFactory,
 ) WorkloadIdentityStorage {
-	workloadIdentityRest := NewREST(optsGetter)
+	issuerURL := ""
+	if tokenIssuer != nil {
```

Comment:
Is this check needed? Can this actually be `nil` here 🤔 I see that we have the following check https://github.com/gardener/gardener/blob/5ad55492db964a460b9a8e4a67cee9e5ad7a8f36/pkg/apiserver/registry/security/workloadidentity/storage/tokenrequest.go#L63-L65

Reply:
Yes, the check is needed because the `--workload-identity-token-issuer` flag is not mandatory and when it is not set, the `tokenIssuer` is `nil`.

Reply:
Got it. Thanks for explaining